.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,5 @@
+-   id: pydocstyle
+    name: Verifying PEP257 Compliance
+    entry: pydocstyle
+    language: python
+    files: \.py$
```

Comment:
`types: [python]` is probably a better approach here

Reply:
cool I will fix it.